packages/extension-sample/src/setup-app.ts

```diff
@@ -26,7 +26,7 @@ class Sample extends ApplicationExtension {
         return [
             {
                 exact: true,
-                path: this.getConfig()?.path || defaultPath,
+                path: this.getConfig().path || defaultPath,
```

Comment:
I deployed a bunlde to MRT https://scaffold-pwa-test-env-3.mobify-storefront.com/ and I see a 500 error.

```
{"message": "Internal server error"}
```
Checking the MRT logs, the error is a type error with `path` :
```
2024-09-18T17:04:05.541Z 8cddec9a ERROR  {'errorType': 'UnhandledPromiseRejection', 'errorMessage': 'TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Array', 'stackTrace': ['Runtime.UnhandledPromiseRejection: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Array', '    at process.<anonymous> (file:///var/runtime/index.mjs:1276:17)', '    at process.emit (node:events:531:35)', '    at process.emit (node:domain:488:12)', '    at emitUnhandledRejection (node:internal/process/promises:250:13)', '    at throwUnhandledRejectionsMode (node:internal/process/promises:385:19)', '    at processPromiseRejections (node:internal/process/promises:470:17)', '    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)'], 'reason': {'errorType': 'TypeError', 'errorMessage': 'The "path" argument must be of type string. Received an instance of Array', 'stackTrace': ['TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Array', '    at Object.join (node:path:1203:7)', '    at /var/task/build/ssr.js:2:9868', '    at Array.forEach (<anonymous>)', '    at Object._setupExtensions (/var/task/build/ssr.js:2:9743)', '    at Object._createApp (/var/task/build/ssr.js:2:6270)', '    at Object.createHandler (/var/task/build/ssr.js:2:16915)', '    at /var/task/build/ssr.js:2:1048689', '    at /var/task/build/ssr.js:2:1048841', '    at Object.<anonymous> (/var/task/build/ssr.js:2:1048880)', '    at Module._compile (node:internal/modules/cjs/loader:1469:14)'], 'code': 'ERR_INVALID_ARG_TYPE'}, 'promise': {}}
```
The type error in the IDE:
```
TS2322: Type unknown is not assignable to type string
```


Plugin configuration in package.json:
```
"mobify": {
    "app": {
      "extensions": [
        [
          "@salesforce/extension-sample",
          {
            "path": "/foo-page",
            "enabled": true
          }
        ]
      ]
    },
```

Reply:
Update: this is working now when deployed.